package.json

```diff
@@ -41,5 +39,9 @@
         "har-validator": "5.x.x",
         "http2": "3.x.x",
         "mocha": "^9.2.0"
+    },
+    "engines": {
```

Comment:
open source library is not restrictive. So, let's follow that and not require this in engines.

Reply:
the build breaks with a version of node lower than 12, this was the motivation to restrict it. If we are against it we should add a .nvmrc with the version of node that we recommend